### PR TITLE
Add link to st2web configuration

### PR DESCRIPTION
**Pull request by trstruth**
The docs mention that some additional configuration may be required to expose st2webui and the services it proxifies, but is rather vague about how that may be done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ kubectl exec -it ${ST2CLIENT} /bin/bash
 st2web is a StackStorm Web UI admin dashboard. By default, st2web K8s config includes a Pod Deployment and a Service.
 `2` replicas (configurable) of st2web serve the web app and proxy requests to st2auth, st2api, st2stream.
 > **Note!** K8s Service uses only NodePort at the moment, so installing this chart will not provision a K8s resource of type LoadBalancer or Ingress (TODO!).
```
Comment (armab) [inline README.md]: We need to rephrase this note as we do allow to configure LoadBalncer as service type for st2web.
Seems we forgot to do that.

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ kubectl exec -it ${ST2CLIENT} /bin/bash
...
-  Depending on your Kubernetes cluster setup you may need to add additional configuration to access the Web UI service or expose it to public net.
+  Depending on your Kubernetes cluster setup you may need to add additional [configuration](https://github.com/StackStorm/stackstorm-ha/blob/master/values.yaml#L218-L219) to access the Web UI service or expose it to public net.
```
Comment (armab) [inline README.md]: I’d prefer to note explicitly a mention that st2web K8s service type could be configured via Helm values to expose the ST2 APIs and Web UI.
That would be better comparing to including a link.

**Comment by armab** [comment]
Additionally, after merging this, we’ll need to update respective st2docs section for K8s st2web 
https://docs.stackstorm.com/install/k8s_ha.html#st2web

File: README.md (hunk 2 above)
Comment (trstruth) [reply README.md]: I agree, the line number reference could be easily broken.  I've updated it.

**Comment by warrenvw** [comment]
Agree it looks better. Thanks guys for doing this!

**Comment by warrenvw** [comment]
I'll update the st2docs.

**Pull request merged**